packages/fluentui/docs/src/examples/components/Menu/Usage/MenuExampleToolbar.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { Menu, menuAsToolbarBehavior } from '@fluentui/react-northstar';
+import {
+  FormatIcon,
+  PaperclipIcon,
+  EmojiIcon,
+  GiphyIcon,
+  StickerIcon,
+  VideoCameraEmphasisIcon,
+  SettingsIcon,
+  MoreIcon,
+  BookmarkIcon,
+  TranslationIcon,
+  TrashCanIcon,
+  MarkAsUnreadIcon,
+} from '@fluentui/react-icons-northstar';
+
+const MenuExampleToolbar = () => (
+  <Menu defaultActiveIndex={0} iconOnly accessibility={menuAsToolbarBehavior} aria-label="Compose Editor">
+    <Menu.Item index={0} aria-label="Format Tool" icon={<FormatIcon outline={true} />} />
```

Comment:
```suggestion
    <Menu.Item index={0} aria-label="Format Tool" icon={<FormatIcon outline />} />
```

And the same for lines below